src/whatthepatch/patch.py

```diff
@@ -914,3 +921,87 @@ def parse_rcs_ed_diff(text):
         return changes
 
     return None
+
+
+def parse_binary_diff(text):
```

Comment:
I think if this only handles git files, it should be named `parse_git_binary_diff`.  I can change this myself before release, or you can if you get back to it before I merge.